src/NuGet.Core/NuGet.Build.Tasks/Strings.resx

```diff
@@ -117,6 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="NoCompatibleTargetFramework" xml:space="preserve">
+    <value>Cannot find a compatible framework for '{0}' in the following frameworks: {1}.</value>
```

Comment:
We should use the same error message as before here:

https://github.com/dotnet/sdk/blob/d7730b73a5bea6c1e7f5d178711322082c93d96e/src/Tasks/Common/Resources/Strings.resx#L124